src/main/java/com/example/b2b_opportunities/Controller/PositionController.java

```diff
@@ -54,4 +56,14 @@ public PositionResponseDto editPosition(@PathVariable("id") Long id, @RequestBod
     public void deletePosition(@PathVariable("id") Long id, Authentication authentication) {
         positionService.deletePosition(id, authentication);
     }
+
+    @PatchMapping("/{id}/status")
+    @ResponseStatus(HttpStatus.OK)
+    public void editPositionStatus(
+            @PathVariable("id") Long id,
+            @Valid @RequestBody ProjectStatusUpdateRequestDto statusUpdateRequestDto,
```

Comment:
Do we need an additional DTO - ProjectStatusUpdateRequestDto? Can we not just use PositionStatus instead? In fact you are only using the ID of the PositionStatus. The name is completely ignored. The only thing needed is a Long - ID of the status

Reply:
I agree with @Yordanofff . We can just pass the status ID that will be set for the Position.